src/bundle/test-transformer.ts

```diff
@@ -22,23 +22,26 @@ export const setupTransformHook = (): VoidFunction => {
                 },
             ],
             require("@babel/plugin-transform-modules-commonjs"),
-            [
-                (): PluginObj => ({
-                    name: "ignore-imports",
-                    visitor: {
-                        ImportDeclaration(path: NodePath<ImportDeclaration>): void {
-                            const extname = nodePath.extname(path.node.source.value);
-
-                            if (extname && !extname.match(JS_EXTENSION_RE)) {
-                                path.remove();
-                            }
-                        },
-                    },
-                }),
-            ],
         ],
     };
 
+    const customIgnoreImportsPlugin = (): PluginObj => ({
```

Comment:
Thinking out loud, I wonder if this was being applied to nodejs code by genuine oversight or for a reason... But since before having CT, everything worked fine without this code, it should work fine now, too

Reply:
I think he just didn't thought ".tsx" files could be read in "nodejs" env, without explicit file endings.
As we in "nodejs" env don't know statically "which extension" the file have, i think it is better to just don't do anything, as it produces hard to debug problems.

I suggest to think about it later, then with CT contributor we could discuss this. For example, we could only remove unnamed imports (like `import "styles.css"`) and don't touch named imports (like `import myModule from "foo.css"`), or have specific blacklist of extensiong (for example, remove `import foo from "styles.css"`, but don't remove `import {foo} from "list.constants"`)